CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
+ * Workspace symbols and CodeLens for Renarde application REST endpoints. See [#777](https://github.com/redhat-developer/quarkus-ls/issues/777).
+ * CodeAction to insert expected `<input>` for Renarde `#form` section in Qute template. See [#778](https://github.com/redhat-developer/quarkus-ls/issues/778).
+ * Show validation errors for all Qute templates in a project, even unopened ones, without first needing to open a template. See [#813](https://github.com/redhat-developer/quarkus-ls/issues/813).
+ * Improve user tag snippet completion in Qute templates. See [#784](https://github.com/redhat-developer/quarkus-ls/issues/784).
+ * Validate user tag parameters in Qute templates. See [#788](https://github.com/redhat-developer/quarkus-ls/issues/788).
+ * Go to definition for user tag parameters in Qute templates. See [#789](https://github.com/redhat-developer/quarkus-ls/issues/789).
+ * CodeAction to insert required user tag parameters in Qute template. See [#797](https://github.com/redhat-developer/quarkus-ls/issues/797).
+ * Context-aware snippets in Java. See [#782](https://github.com/redhat-developer/quarkus-ls/issues/782).
+ * Snippet, documentation, and validation for Qute #fragment section. See [#768](https://github.com/redhat-developer/quarkus-ls/pull/768).
+ * CodeLens, diagnostics, and document links for Qute fragments. See [#769](https://github.com/redhat-developer/quarkus-ls/pull/769).
+ * When applying the Qute template quick fix to generate a missing class member, generate the member at the end of the class. See [#692](https://github.com/redhat-developer/quarkus-ls/pull/692).
+ * Support for validation, completion and document link for `#include` and `#insert` sections in Qute templates. See [#438](https://github.com/redhat-developer/quarkus-ls/issues/438).
+ * "Show references" CodeLens for `#insert` section in Qute templates. See [#814](https://github.com/redhat-developer/quarkus-ls/issues/814).
+ * Support surround with section/comments command for Qute templates. See [#801](https://github.com/redhat-developer/quarkus-ls/pull/801)
+
+### Bug Fixes
+
+ * Treat `CompletionStage` and `Uni` objects as their resolved type for validation and completion in Qute templates. See [#826](https://github.com/redhat-developer/quarkus-ls/issues/826).
+ * `{#if}` sections in Qute templates of the form `{#if !inject:beanName}{/if}` are marked as invalid. See [#828](https://github.com/redhat-developer/quarkus-ls/issues/828).
+ * Quick fix to generate a `@TemplateExtension` class from a Qute template sometimes places the class in the wrong folder. See [#831](https://github.com/redhat-developer/quarkus-ls/issues/831).
+ * Fix ClassCastException in qute template. See [#816](https://github.com/redhat-developer/quarkus-ls/issues/816).
+ * Use quarkus-ls's Qute parser for syntax validation in order to get consistent diagnostic positions and ranges. See [#812](https://github.com/redhat-developer/quarkus-ls/issues/812).

```

Comment:
Could we say "Use the fault tolerant parser ..... " . I assume that would distinguish it as ours ? Also should this be under enhancements maybe ?

Reply:
The reason I listed it here was because Angelo made this change to fix a bug where the error range for an errror made no sense and caused a bug in code actions.